pkg/config/manifest.go

```diff
@@ -446,6 +446,7 @@ func (c *Config) readFromFile(path string) (*Manifest, error) {
 	if err != nil {
 		return nil, err
 	}
+	m.SetDefaults()
```

Comment:
I'm wondering if this call should be higher up (say, in `ReadManifest()`?)  This would set a default image regardless of if we are reading from a file or a URL, then.  Does that sound like a valid case as well?

Reply:
Yes it does, but does the version tag of the invocationImage default to the version tag of `manifest.Tag` or the `manifest.Version` or just `latest`?

Reply:
Ah, great question!  Let's check with @carolynvs on precedence here.  I'm still unsure on which would be canonical to represent the bundle version.

Reply:
@phillipahereza It looks like we want to default to `manifest.Version` for the version tag.